custom_components/octopus_intelligent/manifest.json

```diff
@@ -7,5 +7,5 @@
   "codeowners": ["@megakid"],
   "iot_class": "cloud_polling",
   "config_flow": true,
-  "requirements": ["gql==3.2.0"]
+  "requirements": ["gql~=3.5.0"]
```

Comment:
* `gql` version 3.2.x does not “officially” support Python versions 3.11 or 3.12, which are currently required by HASS. `gql` version 3.5.0 [does](https://github.com/graphql-python/gql/releases/tag/v3.5.0).
* In case `gql` had been pinned to version 3.2.0 because of specific issues with versions 3.3.x or 3.4.x, it may be worth noting that in version 3.5.0, [_“graphql-core has been reverted to the stable versions 3.2.x”_](https://github.com/graphql-python/gql/releases/tag/v3.5.0) (also see [this comment](https://github.com/graphql-python/gql/issues/427#issuecomment-1875526451)).
* `gql` version 3.2.0 was release almost two years ago and would need to be updated at some point, and I thought that now might as well be the time.
* `~=3.5.0` means 3.5.*, allowing HASS to choose versions 3.5.1, 3.5.2 and so on (the latest version available) when the integration is installed.
* I tested the integration with this change and HASS version 2024.2.5 and basically I did not notice any difference! Importantly, nothing broke.